lib/terraforming/resource/efs.rb

```diff
@@ -0,0 +1,41 @@
+# author: Jim Conner <snafu.x@gmail.com>
+# Nov 2016
+module Terraforming
+  module Resource
+    class EFS
+      include Terraforming::Util
+
+      def self.tf(client: Aws::EFS::Client.new)
+        self.new(client).tf
+      end
+
+      def self.tfstate(client: Aws::EFS::Client.new)
+        self.new(client).tfstate
+      end
+
+      def initialize(client)
+        @client = client
+      end
+
+      def tf
+        apply_template(@client, "tf/elastic_filesystem")
+      end
+
+      def tfstate
+        efsystems.inject({}) do |resources, efs|
+          attributes = {
```

Comment:
__[rubocop]__
* Useless assignment to variable - `attributes`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->